editoast/openapi.yaml

```
@@ -1324,6 +1324,39 @@ paths:
                         train_path:
                           type: number
 
+  /timetable/{id}/conflicts/:
+    get:
+      tags:
+        - timetable
```

```suggestion
      tags:
        - timetable
```
Are you gonna refresh this url ? The tags should be used only if we are going to invalidate the result at a moment

---

I think it is okay like this !